src/wx/CMakeLists.txt

```diff
@@ -1008,7 +1008,7 @@ if(NOT TRANSLATIONS_ONLY)
         endif()
     elseif(MSVC)
         # the debug lib libcmtd is linked in debug mode, so don't link the normal version
-        set_target_properties(visualboyadvance-m PROPERTIES LINK_FLAGS_DEBUG "/nodefaultlib:libcmt /subsystem:console")
+        target_link_options(visualboyadvance-m PRIVATE "$<$<CONFIG:DEBUG>:/nodefaultlib:libcmt;/subsystem:console>")
```

Comment:
This (/nodefaultlib) typically is an indicator that something is linking wrongly.

Reply:
its required to to make use of Xaudio from the DXSDK rather than Windows SDK, as the former uses Xaudio2.7, rather than 2.8 which is only available in Windows 8+.

The alternative is to use Xaudio2.9 redist but that adds loose files to the package.